### PR TITLE
Adds jupyter notebook version of multi-ship loading sample, plus a readme

### PR DESCRIPTION
**Pull request by frtibble**
### Summary
This adds a jupyter notebook version of the sample contributed by @ffrachon, and a README to the folder.

### Open Issues
The use of 'combinations' to generate all the terms is not explained.

### CODE REVIEW

**Comment by anraman** [comment]
@frtibble it would also be good to add a link to this new sample on the homepage readme where the other samples are listed

**Comment by frtibble** [comment]
> @frtibble it would also be good to add a link to this new sample on the homepage readme where the other samples are listed

Done!

**Comment by ffrachon** [comment]
All looks good to me at this point!

**Review by ffrachon — DISMISSED** [review]

File: samples/multiship-loading-sample/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+This sample provides a comprehensive walkthrough of the multi-ship loading problem, from multiple sub-problem definitions to using the Azure Quantum Optimization Service to obtain results and help choose solver parameters. This is an intermediate-level sample.
+
+By working through this sample, you will learn:
+
+- How to tackle the multi-ship loading problem
+- How to define two sub cost-functions and transform these mathematical functions into code using the Azure Quantum Optimization Python SDK
+- How to submit problem terms to Azure Quantum
+- How to obtain parameters returned from the parameter-free solvers for use with other parametrized solvers
+
+Before approaching this sample, it is recommended that you complete the ship-loading sample, which can be found in this repository.
```
Comment (anraman) [inline samples/multiship-loading-sample/README.md:27]: Would be good to link to the folder with the ship loading sample :)

**Review by anraman — CHANGES_REQUESTED** [review]
I am going to create a PR from my own branch into this PR branch to address some of the larger changes to the maths in the first section as it will be a lot easier to review that way. If you agree with the minor code changes I have suggested, it would also be good to add these modifications to the python script.

**Comment by frtibble** [comment]
Thanks @anraman - ready for your review

**Review by anraman — APPROVED** [review]

**Pull request merged**